e2e/nomostest/new.go

```diff
@@ -424,6 +424,7 @@ func setupTestCase(nt *NT, opts *ntopts.New) {
 	if err := waitForConfigSync(nt, opts.Nomos); err != nil {
 		nt.T.Fatalf("waiting for ConfigSync Deployments to become available: %v", err)
 	}
+	createCentralRootSync(nt)
```

Comment:
Why do we need to create a central RootSync here and below?

Reply:
It was inside the `waitForConfigSync` function and then I extracted it out in the last patchset. 
Now, I reverted the change. 

The central RootSync is used to bootstrap other RSyncs in the central-control mode.